packages/server-core/src/projects/scene/scene.hooks.ts

```diff
@@ -217,8 +218,12 @@ const createSceneInStorageProvider = async (context: HookContext<SceneService>)
     )
   )
   try {
-    await context.storageProvider.createInvalidation(
-      SCENE_ASSET_FILES.map((asset) => `${data.directory}${context.newSceneName}${asset}`)
+    await Promise.all(
```

Comment:
Instead of making multiple requests to invalidation service. Pass array of {path: ''} object in 1 create call.